build/media-suite_helper.sh

```diff
@@ -220,7 +220,15 @@ do_vcs() {
     cd_safe "$LOCALBUILDDIR"
     if [[ ! -d "$vcsFolder-$vcsType" ]]; then
         do_print_progress "  Running $vcsType clone for $vcsFolder"
-        log quiet "$vcsType.clone" vcs_clone || do_exit_prompt "Failed cloning to $vcsFolder-$vcsType"
+        if ! log quiet "$vcsType.clone" vcs_clone && [[ $vcsType = git ]] &&
+            [[ "${vcsURL:19:34}" != "media-autobuild_suite-dependencies" ]]; then
```

Comment:
Maybe safer to use pattern matching instead of hardcoded character positions.

```suggestion
            [[ "$vcsURL" != *"media-autobuild_suite-dependencies"* ]]; then
```